__tests__/app.test.js

```diff
@@ -163,3 +163,60 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("should respond with a 201 and an object of the newly posted comment", () => {
+    const newComment = {
+      username: "mallionaire",
+      body: "amazing game for the family",
+    };
+    return request(app)
+      .post("/api/reviews/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          postedComment: { author: "mallionaire", body: "amazing game for the family" },
+        });
+      });
+  });
```

Comment:
Please add another test, where unnecessary properties are ignored when posting a new comment. For example when posting with `{author: "mallionaire", body: "A brand new comment.", is_best_comment: "true"}`. Expectiing 201 and comment including only the 'author' and 'body' properties.